### PR TITLE
[GRPH-137] Merge gpos into develop

### PR DESCRIPTION
**Pull request by oxarbitrage**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -9,6 +9,8 @@ RUN \
     apt-get update -y && \
       DEBIAN_FRONTEND=noninteractive apt-get install -y \
       autoconf \
+      gcc-5 \
+      g++-5 \
```
Comment (pbattu123) [inline Dockerfile]: These changes line 12 and 13 are not required. The build is compatible with gcc 7 also

File: Dockerfile [hunk 2]
```diff
@@ -50,6 +52,7 @@ WORKDIR /peerplays-core
 # Compile Peerplays
 RUN \
     BOOST_ROOT=$HOME/boost_1_67_0 && \
+    export CC=gcc-5 ; export CXX=g++-5\
```
Comment (pbattu123) [inline Dockerfile]: These changes are not required. The build is compatible with gcc 7 too

File: Dockerfile (hunk 1 above)
Comment (oxarbitrage) [reply Dockerfile]: thanks. removed at https://github.com/peerplays-network/peerplays/pull/212/commits/a07bcad7e80877b2a63fff73d1175c9f7ae477af

File: Dockerfile (hunk 2 above)
Comment (oxarbitrage) [reply Dockerfile]: https://github.com/peerplays-network/peerplays/pull/212/commits/a07bcad7e80877b2a63fff73d1175c9f7ae477af

**Pull request merged**